hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -14,6 +14,8 @@ parameters:
   required: true
 - name: ESCALATION_POLICY
   required: true
+- name: ALERTS_SKIP_LEGALENTITY_IDS
+  value: '["NOOP"]'
```

Comment:
This can't just be a blank array?

Reply:
probably, don't remember why I did this.  similar in https://github.com/openshift/splunk-forwarder-operator/pull/85

Reply:
If we can make it blank its less confusing since `NOOP` doesn't mean anything.